app/models/task.py

```diff
@@ -1,5 +1,48 @@
 from app import db
+from datetime import datetime
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True )
+    title = db.Column(db.String, nullable=False)
+    description = db.Column(db.String, nullable=False)
+    completed_at = db.Column(db.DateTime, nullable=True, default=None)
```

Comment:
In this situation, the default for nullable is True, so we do not have to specify that particular constraint! Similarly, when nullable is set to True, the default for default is None, so that also doesn't need to be specified!